framework/tlvf/AutoGenerated/include/tlvf/ieee_1905_1/tlvDeviceInformation.h

```diff
@@ -35,25 +37,13 @@ class tlvDeviceInformation : public BaseClass
         tlvDeviceInformation(std::shared_ptr<BaseClass> base, bool parse = false);
         ~tlvDeviceInformation();
 
-        typedef struct sInfo {
-            sMacAddr mac;
-            eMediaType media_type;
-            uint8_t media_info_length;
-            eMediaType media_info;
-            void struct_swap(){
-                mac.struct_swap();
-            }
-            void struct_init(){
-                mac.struct_init();
-            }
-        } __attribute__((packed)) sInfo;
-        
         const eTlvType& type();
         const uint16_t& length();
         sMacAddr& mac();
-        uint8_t& info_length();
-        std::tuple<bool, sInfo&> info(size_t idx);
-        bool alloc_info(size_t count = 1);
+        uint8_t& local_interface_list_length();
```

Comment:
Seems to me that this should be read-only, I don't know what you gain by exposing this

Reply:
This is auto-generated code